File: LCM.py
def lcm(a, b):
    max_val = a if a > b else b
    lcm_val = max_val
    while True:
        if lcm_val % a == 0 and lcm_val % b == 0:
            return lcm_val
        lcm_val += max_val

x = int(input("Enter first integer: "))
y = int(input("Enter second integer: "))
print(f"LCM of {x} and {y} is {lcm(x, y)}")

File: LCM.java
import java.util.Scanner;
public class LCM {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        System.out.print("Enter first integer: ");
        int a = input.nextInt();
        System.out.print("Enter second integer: ");
        int b = input.nextInt();

        int max = (a > b) ? a : b;
        int lcm = max;

        while (true) {
            if (lcm % a == 0 && lcm % b == 0) {
                break;
            }
            lcm += max;
        }

        System.out.println("LCM of " + a + " and " + b + " is " + lcm);
    }
}